api/src/api/v1/community_businesses/volunteer_logs.ts

```diff
@@ -362,11 +367,11 @@ const routes: Hapi.ServerRoute[] = [
 
       // Ignore invalid logs silently because of
       // https://github.com/TwinePlatform/twine-monolith/issues/246
-      const payload = ignoreInvalidLogs(_payload);
+      const { valid: payload, invalid } = ignoreInvalidLogs(_payload);
```

Comment:
why is this method only used in the `sync` handler, couldn't this error happen on POST/PUT methods too?

Reply:
I'm not sure because I haven't spent any time looking into why this timestamp corruption might have happened. In principle, if the timestamp is corrupted when adding/editing, the app would fall into offline mode, and the user would have to sync.